libs/libc/machine/arm/gnu/arch_setjmp.S

```diff
@@ -31,7 +31,9 @@
 	.globl	setjmp
 	.globl	longjmp
 
+#ifndef __ghs__
 	.syntax	unified
```

Comment:
you could define "`.syntax	unified`" to empty to avoid change common code

Reply:
> you could define "`.syntax unified`" to empty to avoid change common code

Ok, I will try this in a new patch